server/game/Effects/Effect.js

```diff
@@ -88,6 +88,9 @@ class Effect {
         if(this.duration !== Durations.Persistent) {
             return true;
         }
+        if(this.source.type === 'token') {
+            return true;
+        }
         let effectOnSource = this.source.persistentEffects.some(effect => effect.ref && effect.ref.includes(this));
```

Comment:
Sure, this was a bit of hack to get it to work, but yeah I suppose tokens are never 'facedown'.  Nice idea about Yojiro, that may save having the HonorStatusDoesNotModifySkill effect.  I suggest that is saved for another PR though.

Reply:
Ah yeah, I put this in because the source (the token) does not have any persistent effects.  I think I will try and reverse all of this and have a go at adding the persistent effects directly on to the card.